cypress/integration/homepage.spec.ts

```diff
@@ -1,5 +1,6 @@
 describe('Home page', () => {
   beforeEach(() => {
+    cy.clearStorage();
```

Comment:
Cypress clears storage by default before every single test
From: https://docs.cypress.io/api/commands/clearlocalstorage

> Cypress automatically runs this command before each test to prevent state from being shared across tests. You shouldn't need to use this command unless you're using it to clear localStorage inside a single test.

Reply:
that's weird because when I was testing it locally, it was preserving the state between the tests